core/src/main/scala/sttp/client3/logging/Log.scala

```diff
@@ -61,13 +61,24 @@ class DefaultLog[F[_]](
       }
     )
 
-  override def requestException(request: Request[_, _], elapsed: Option[Duration], e: Exception): F[Unit] =
-    logger(responseExceptionLogLevel, s"Exception when sending request: ${request.showBasic}${took(elapsed)}", e)
+  override def requestException(request: Request[_, _], elapsed: Option[Duration], e: Exception): F[Unit] = {
+    val logLevel = e match {
+      case HttpError(_, statusCode) =>
+        responseExceptionLogLevel(Some(statusCode))
```

Comment:
maybe here we should use simply `responseLogLevel`?

Reply:
Yes, but what to do when  statusCode is not available?

Reply:
Then we use `responseExceptionLogLevel`? as we are dealing with an exception which prevented the response from being obtained

Reply:
Yes, actually this was my initial solution. But I was not sure if we can reuse responseLogLevel for handling HttpError (which is an Exception) while providing responseExceptionLogLevel. That perhaps could be confusing.

Reply:
Yeah I was not sure either, but then `HttpError` is an "artificially" introduced exception to represent unwanted status codes, not something that comes from the backend, so maybe this will be less confusing than two log level configurations, both accepting status codes? :)

Reply:
Not sure I follow your reasoning,  but I agree :)

Reply:
I expected responseLogLevel function (that accepts StatusCode) to be used for handling HttpError. Later I was suprised it does not work as I thought. So yes, I agree with you @adamw .

Reply:
I don't know why some tests failed.